gguf-py/gguf/tensor_mapping.py

```diff
@@ -919,6 +920,11 @@ class TensorNameMap:
             "vpm.embeddings.patch_embedding",
             "model.vision_model.embeddings.patch_embedding", # SmolVLM
             "vision_tower.patch_conv", # pixtral
+            "visual.patch_embed.proj", # qwen2vl
+        ),
+
+        MODEL_TENSOR.V_ENC_EMBD_PATCH1: (
+            "visual.patch_embed.proj.weight.1", # qwen2vl, generated
```

Comment:
Is this going to work correctly - usually the `".weight"` is suffixed to the base name, so wouldn't this try to map `"visual.patch_embed.proj.weight.1.weight"`?

Reply:
The logic of `map_tensor_name` will firstly try mapping with the whole name (which is the case here). If it cannot find a match, then it will retry with `.weight` and `.bias` (which is the case for all other tensors)

Reply:
hmm on second thought, there is a way to confine this hacky logic inside `Qwen2VLVisionModel`, so it won't pollute the rest of the code. I did it in https://github.com/ggml-org/llama.cpp/pull/13209/commits/c030984b5a30eb2e5530516bfd7c4b8a42dbcf52

Here is the output:

```
INFO:hf-to-gguf:v.patch_embd.weight,           torch.bfloat16 --> F16, shape = {14, 14, 3, 1280}
INFO:hf-to-gguf:v.patch_embd.weight.1,         torch.bfloat16 --> F16, shape = {14, 14, 3, 1280}
```

Reply:
Nice, much better